index.js

```diff
@@ -11,68 +11,145 @@ module.exports = {
     'plugin:vue/recommended'
   ],
   plugins: [
-    'vue',
-    'jest'
+    'jest',
+    'unicorn',
+    'vue'
   ],
   settings: {
     'import/resolver': {
       node: { extensions: ['.js', '.mjs'] }
     }
   },
   rules: {
+
+    /**********************/
+    /* General Code Rules */
+    /**********************/
+
     // Enforce import order
-    'import/order': 2,
+    'import/order': 'error',
 
     // Imports should come first
-    'import/first': 2,
+    'import/first': 'error',
 
     // Other import rules
-    'import/no-mutable-exports': 2,
+    'import/no-mutable-exports': 'error',
 
     // Allow unresolved imports
-    'import/no-unresolved': 0,
+    'import/no-unresolved': 'off',
 
     // Allow paren-less arrow functions only when there's no braces
-    'arrow-parens': [2, 'as-needed', { requireForBlockBody: true }],
+    'arrow-parens': ['error', 'as-needed', { requireForBlockBody: true }],
 
     // Allow async-await
-    'generator-star-spacing': 0,
+    'generator-star-spacing': 'off',
 
     // Allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
-    'no-console': process.env.NODE_ENV === 'production' ? 2 : 0,
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
     // Prefer const over let
-    'prefer-const': [2, {
+    'prefer-const': ['error', {
       'destructuring': 'any',
       'ignoreReadBeforeAssign': false
     }],
 
     // No single if in an "else" block
-    'no-lonely-if': 2,
+    'no-lonely-if': 'error',
 
     // Force curly braces for control flow,
     // including if blocks with a single statement
-    curly: [2, 'all'],
+    curly: ['error', 'all'],
 
     // No async function without await
-    'require-await': 2,
+    'require-await': 'error',
 
     // Force dot notation when possible
-    'dot-notation': 2,
+    'dot-notation': 'error',
 
-    'no-var': 2,
+    'no-var': 'error',
 
     // Force object shorthand where possible
-    'object-shorthand': 2,
+    'object-shorthand': 'error',
 
     // No useless destructuring/importing/exporting renames
-    'no-useless-rename': 2,
+    'no-useless-rename': 'error',
+
+    /**********************/
+    /*   Unicorn Rules    */
+    /**********************/
+
+    // Pass error message when throwing errors
+    'unicorn/error-message': 'error',
+
+    // Uppercase regex escapes
+    'unicorn/escape-case': 'error',
+
+    // Use "new" for most builtins (not String, Number, ..., though)
+    'unicorn/new-for-builtins': 'error',
+
+    // Array.isArray instead of instanceof
+    'unicorn/no-array-instanceof': 'error',
+
+    // for-of instead traditional for loop
+    'unicorn/no-for-loop': 'error',
```

Comment:
Maybe this could be a warning due to the large polyfill it requires for older browsers?

Reply:
Why a polyfill? 🤔 
With babel in place, this will be rewritten with almost 0 cost:

https://babeljs.io/repl/#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=MYewdgzgLgBAhjAvDA2gRgDQCYMGYC6AUIQGYgBOMAFGDCCfAJQwDehMMokIANgKYA6HiADmVOI0IBfIA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Creact%2Cstage-2&prettier=false&targets=&version=7.4.3&externalPlugins=